src/js/ProductDetails.mjs

```diff
@@ -0,0 +1,63 @@
+import {
+  addToLocalStorage
+} from "./utils.mjs";
+
+export default class productDetails {
+  constructor(productId, dataSource) {
+    this.productId = productId;
+    this.product = {};
+    this.dataSource = dataSource;
+  }
+
+  async init() {
+    this.product = await this.dataSource.findProductById(this.productId);
+    this.renderProductDetails(this.product);
+    document.getElementById('addToCart')
+      .addEventListener('click', this.addToCart.bind(this));
+  }
+
+  addToCart(product) {
+    addToLocalStorage("so-cart", product);
+  }
+
+  renderProductDetails(product) {
+    document.pageTitle = `Sleep Outside | ${product.Name}`
+    const section = document.querySelector(".product-detail")
+    section.innerHTML = `<h3>${product.Name}</h3>
+
+    <h2 class="divider">${product.NameWithoutBrand}</h2>
+
+    <div class="divider">
+      <picture>
+        <source
+          media="(min-width:720px)"
+          srcset="${product.Image}"
+        />
+        <source
+          media="(min-width:540px)"
+          srcset="${product.Image.replace("320","240")}"
+        />
+        <source
+          media="(min-width:360px)"
+          srcset="${product.Image.replace("320","160")}"
+        />
+        <img
+          src="c"
```

Comment:
I think you accidentally deleted the src attribute here? Can you verify that this works?

Reply:
Yep, sorry, I did delete it unintentionally. Thanks for spotting it